include/lsst/afw/image/TransmissionCurve.h

```diff
@@ -0,0 +1,300 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST/AURA.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_AFW_IMAGE_TransmissionCurve_h_INCLUDED
+#define LSST_AFW_IMAGE_TransmissionCurve_h_INCLUDED
```

Comment:
I actually like this convention much better, because it's compatible with case-sensitive and underscore-invariant search-and-replace.  But it's not what's in the dev guide, and it's not worth effort to change that.  I'll change the case here.